### PR TITLE
Last updated text for charts

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-110

Adds the years listed in the spreadsheet.

### CODE REVIEW

File: src/components/CityColumnChart/CityColumnChart.js [hunk 1]
```diff
@@ -5,14 +5,15 @@ import Highcharts from 'highcharts';
 import kebabCase from 'lodash/kebabCase';
 import merge from 'lodash/merge';
 import numeral from 'numeral';
-import AboutTooltip from '../AboutTooltip/AboutTooltip';
```
Comment (davidgilbertson) [inline src/components/CityColumnChart/CityColumnChart.js:8]: I ditched this, it does nothing and now there's two `<Tooltip>`s in the chart, it makes no sense to have a special component for one tooltip but not the other.

File: src/components/CityColumnChart/CityColumnChart.js [hunk 2]
```diff
@@ -85,23 +86,16 @@ function getChartConfig(props) {
...
   // so here we take the passed in value, or the value from the first indicator otherwise.
   const firstIndicator = INDICATORS[props.indicatorIds[0]];
-  const shortDescription = props.shortDescription || firstIndicator.shortDescription;
   const data = sortChartData(props.cities, props.indicatorIds[0]);
   const plotBands = getPlotBands(data, props.city, colorLight);
   const series = getSeries(props);
 
-  // We only want to show the short description as the chart title
-  // if the chart is not stacked
-  const yAxisTitle = isMultiple ? {} : { text: shortDescription };
```
Comment (davidgilbertson) [inline src/components/CityColumnChart/CityColumnChart.js:95]: I've done some aligning of all the top elements (title, "last updated" legend and axis label). This includes always showing the axis label up the top, and never as part of the high chart.

File: src/components/CityColumnChart/CityColumnChart.js [hunk 3]
```diff
@@ -144,7 +138,7 @@ function getChartConfig(props) {
       },
     },
     yAxis: {
-      ceiling,
+      ceiling: props.stacked ? 1 : null,
```
Comment (davidgilbertson) [inline src/components/CityColumnChart/CityColumnChart.js:141]: Just moving stuff around for no good reason.

File: src/constants/indicators.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ export default {
     longDescription: 'This shows the population size.\n\nSource: ABS - Regional Population Growth',
     format: '0[.]0a',
     contextual: true,
+    lastUpdated: '2016',
```
Comment (micapam) [inline src/constants/indicators.js:15]: The designs show a full date, e.g. `27 July 2016`, rather than just the year. Was that data not available?

**Review by micapam — APPROVED** [review]

**Pull request merged**